_data/meltano/loaders/target-mongodb/hotgluexyz.yml

```
@@ -0,0 +1,27 @@
+name: target-mongodb
+variant: hotgluexyz
+label: MongoDB
+logo_url: /assets/logos/loaders/mongodb.png
+capabilities:
+- about
+description: MongoDB
+domain_url: https://www.mongodb.com/
+keywords:
+- meltano_sdk
+- database
+namespace: target_mongodb
+pip_url: git+https://gitlab.com/hotglue/target-mongodb.git
+repo: https://gitlab.com/hotglue/target-mongodb
+settings:
+- name: connection_string
+  label: Connection String
+  description: Connection string in the format mongodb://[${MONGO_USERNAME:${MONGO_PASSWORD}@]mongodb0.example.com:PORT
+  kind: string
```

```suggestion
  kind: password
```

@sbalnojan what do you think about making this a `password` type? I think many users would put their user/pass in there directly which would expose creds into the meltano.yml. I found a few other cases like airflow/superset where we allow a connection string as a string type but might be worth considering. [target-postgres](https://github.com/meltano/hub/blob/4eafa14a013b762342d2c5766465b22205c4d999/_data/meltano/loaders/target-postgres/meltano.yml#L45) example

---

puh, I agree with you. We should advocate for data best practices, and there I believe in need to know and data scarcity. So better err on the cautious side here.